### PR TITLE
Specify virbr2 for bridge and don't install libvirt-daemon-config-net…

### PR DESCRIPTION
**Pull request by p3ck**
…work

which has the default config of virbr0

### CODE REVIEW

**Comment by p3ck** [comment]
[test]

**Comment by p3ck** [comment]
[test]

**Comment by p3ck** [comment]
[test]

File: config/Dockerfiles/ostree-boot-image/default.xml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <network>
   <name>default</name>
+  <uuid>03f5e3d1-164e-4059-9df8-8c1494b81025</uuid>
```
Comment (johnbieren) [inline config/Dockerfiles/ostree-boot-image/default.xml]: do these uuids ever have to change?

Comment (p3ck) [reply config/Dockerfiles/ostree-boot-image/default.xml]: no - and they could probably even be the same between containers.

**Comment by samvarankashyap** [comment]
@p3ck @johnbieren : Is there any way we could spin up the containers when we need and delete them when it finishes it job instead of running them simultaneously. Which might solve the our issue. Further it uses the less resources at any given point of time . ( I don't know whether it's applicable or not in our use case . Just a thought )

**Comment by johnbieren** [comment]
@samvarankashyap my understanding was that the whole benefit of using OpenShift is that the containers are always up and available so you don't have to waste time bringing them up before using them. I am far from an SME on that front, though.

**Comment by samvarankashyap** [comment]
@johnbieren : I completely agree with you . However, I also observed that now that we are building the images and storing them in openshift registry . It hardly takes seconds to boot up the containers from the image in openshift . If the images are pulled down from external registries like docker , it would take time though.

**Comment by p3ck** [comment]
[test] disk actiity timeout

**Comment by p3ck** [comment]
[merge]

**Comment by p3ck** [comment]
[merge]

**Pull request merged**